lightwood/api/json_ai.py

```diff
@@ -26,7 +26,7 @@ def lookup_encoder(
         dtype.image: 'Image.Img2VecEncoder',
         dtype.rich_text: 'Rich_Text.PretrainedLangEncoder',
         dtype.short_text: 'Short_Text.CategoricalAutoEncoder',
-        dtype.array: 'Array.TimeSeriesEncoder',
+        dtype.array: 'Array.ArrayEncoder',
```

Comment:
ts_array needs to go in here as well with an addition to `lw.api.dtype`.

Reply:
@paxcema

Reply:
But it needn't be part of this PR... I think